packages/core/src/components/_index.scss

```diff
@@ -89,6 +89,48 @@ Styleguide components.usage.npm
 */
 
 /*
+CDN consumption
+
+Blueprint supports the venerable [unpkg CDN](https://unpkg.com). Each package provides a UMD
+`dist/[name].bundle.js` file containing the bundled source code. The UMD wrapper exposes each
+library on the `Blueprint` global variable: `Blueprint.Core`, `Blueprint.Datetime`, etc.
+
+These bundles _do not include_ external dependencies; your application will need to ensure that
+`normalize.css`, `React`, `classnames`, and `Tether` are available at runtime.
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Blueprint Starter Kit</title>
+    <link href="https://unpkg.com/normalize.css@5.0.0" rel="stylesheet" />
+    <link href="https://unpkg.com/@blueprintjs/core@1.11.0/dist/blueprint.css" rel="stylesheet" />
+  </head>
+  <body>
+    <script src="https://unpkg.com/classnames@2.2.5"></script>
```

Comment:
these version numbers will quickly get outdated. how about using semver ranges and then we only have to update these script tags once every major release?